src/KafkaFlow.UnitTests/BatchConsume/BatchConsumeMiddlewareTests.cs

```diff
@@ -37,8 +37,8 @@ public async Task Invoke_DifferentWorkers_CallAddAsyncForEachOne()
             var contextWorker1Mock = new Mock<IMessageContext>();
             var contextWorker2Mock = new Mock<IMessageContext>();
 
-            var consumerContextWorker1Mock = new Mock<IMessageContextConsumer>();
-            var consumerContextWorker2Mock = new Mock<IMessageContextConsumer>();
+            var consumerContextWorker1Mock = new Mock<IConsumerContext>();
```

Comment:
I think we could adopt the standard 'type variable = new()' instead of 'var variable = new type()'

Reply:
When we have a tool to change the entire solution to use the new(), we can adopt it as a guideline. Before that, let's use it for class attributes only.